docs/source/AdministratorGuide/Systems/Accounting/index.rst

```diff
@@ -39,3 +39,22 @@ Since v6r12 each accounting type can be stored in a different DB. By default all
     }
     
 With the previous configuration all accounting data will be stored and retrieved from the usual database except for the _WMSHistory_ type that will be stored and retrieved from the _Acc2_ database.
+
+DataStore Helpers
+======================
+From DIRAC v8r17p14 we are able to run multiple services. The master will creates the proper buckets and the helpers only insert the records to the 'in' table.
```

Comment:
I'd rephrase to 

"From DIRAC v6r17p14 there is the possibility to to run multiple 'DataStore' services, where one need to be called 'master', while all the others may be called slaves. The master will creates the proper buckets and the helpers only insert the records to the 'in' table."